sgx-tools/gen-qe-target-info.go

```diff
@@ -17,12 +18,20 @@ For example, save the target information file about Quoting Enclave retrieved fr
 
 	# sgx-tools gen-qe-target-info --targetinfo foo`,
 	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "isDCAP",
```

Comment:
I hate "isDCAP". It means you need to define isBAR, isFOO and so on? Defining a general attestation type is better.